Modules/Loadable/InteractiveTubesToTree/Widgets/qSlicerInteractiveTubesToTreeTableWidget.cxx

```diff
@@ -430,6 +495,52 @@ bool qSlicerInteractiveTubesToTreeTableWidget::getTubeDisplayColor(QColor &TubeD
   return true;
 }
 
+// --------------------------------------------------------------------------
+bool qSlicerInteractiveTubesToTreeTableWidget::getTubeRootStatus(int row)
```

Comment:
I'm not sure about whther this function is useful. It is called only once, the caller passes int row and this function re-retrieves only data the caller already has. The caller could simply call the return statement directly, couldn't it? The same probably applies to getTubeParentId. But I may be wrong in both cases.